### PR TITLE
Added vendor/project namespace, restructured folders for PSR-4 compat…

### PR DESCRIPTION
**Pull request by hollodotme**
…ibility

* Updated autoload directives in composer.json.
* No functional changes.

Related to #54

### CODE REVIEW

**Comment by localheinz** [comment]
Related to https://github.com/codeclimate/php-test-reporter/pull/35.

File: composer.json [hunk 1]
```diff
@@ -1,37 +1,57 @@
 {
-    "name": "codeclimate/php-test-reporter",
-    "description": "PHP client for reporting test coverage to Code Climate",
```
Comment (localheinz) [inline composer.json]: @hollodotme 

The reformatting here seems unrelated, doesn't it?

Comment (hollodotme) [inline composer.json]: The reformatting, yes, but not the additions at the bottom.
Sorry, My PhpStorm settings differ from the previous indention.

**Comment by pbrisbin** [comment]
This was merged as 4071b3b26443a83983740002f9aa8123fa72e522.
